### PR TITLE
JA: add lite/guide/roadmap.md

### PR DESCRIPTION
**Pull request by mzwtjp**
Originally from:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/guide/roadmap.md

### CODE REVIEW

**Comment by mzwtjp** [comment]
@googlebot I signed it!

File: site/ja/lite/guide/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+私たちはロードマップを4つの分野に区分けしています：
+有用性、性能、最適化、そして可搬性。
+読者はぜひ私たちのロードマップを論評し、
+TF Liteの議論グループもしくはフォーラムでフィードバックしてください。
+
+## 有用性
+
+*   **新しいモデルコンバーター**
+    *   グラフ変換(例、コントロールフロー、条件付き、など)をより上手く行う新しいMLIRベースのTensorFlow Liteコンバーター
+    *   モデル変換の失敗についての診断やデバッグの改善
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: [Weak Suggestion]

`モデル変換の失敗についての` -> `モデル変換失敗時の`

File: site/ja/lite/guide/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+*   **拡張されたopsカバレッジ**
+    *   ユーザーフィードバックに基づいた優先度でのopの追加
+*   **TensorFlowのopsをTensorFlow Liteで使う際の改善**
+    *   バイナリ(Android)やCocoapods(iOS)にて利用できるプリビルドのライブラリ
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: Original sentence says `Bintray` not `Binary`.
I think `Bintray (Android)` is correct here.

File: site/ja/lite/guide/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    *   op strippingによってselect TF opsを使う際のバイナリサイズをより小さく
+*   **LSTM / RNNサポート**
+    *   LSTMとRNNの変換のフルサポート
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: [Weak Suggestion]

`LSTMとRNNの変換の` -> `LSTMとRNNの変換を`

File: site/ja/lite/guide/roadmap.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+*   **前処理と後処理のサポート**
+    *   モデル固有の前処理と後処理のための新しいサポートライブラリ
+    *   共通するプラットフォーム固有機能のためのユーティリティ、例、効率的にassetsからモデルをロードしたりBitmapをテンソルに変換する
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: [Weak Suggestion]

* `例` -> `例えば`
* `assets` -> `アセット`
* `Bitmap` -> `ビットマップ`

File: site/ja/lite/guide/roadmap.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+*   **制御フローとオンデバイスの学習**
+    *   制御フロー関連のopsのサポート
+    *   オンデバイスの学習のサポート、特にパーソナライズ転移学習関連
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: [Weak Suggestion]

I think `personalization` and `transfer learning` should not be together.

File: site/ja/lite/guide/roadmap.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+*   **グラフの可視化ツール類**
+    *   強化したグラフ可視化ツール類の提供
+*   **より多くのモデルと例**
+    *   サイトのサポートセクションにもっとモデルを
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: [Weak Suggestion]

`もっとモデルを` -> `より多くのモデルを追加する`

File: site/ja/lite/guide/roadmap.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+    *   モデルの使用法と新しい機能やAPIをデモするための、各種のプラットフォームをカバーした例を追加。
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: [Weak Suggestion]

`new features` and `APIs`, `covering different platforms` seems to be parallel things in the original text.

File: site/ja/lite/guide/roadmap.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+    *   初心者が容易にカスタマイズできるためのモデルのカスタマイズ用のライブラリとチュートリアル。
+
+## 性能
+
+*   **より良いツール化**
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: `ツール類` is used for the translation about `tooling` above sentence.
We should use `ツール類` here to consistent.

File: site/ja/lite/guide/roadmap.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+    *   利用できるアクセラレーターと性能のトレードオフを理解するための、より簡単なベンチマークとプロファイルツール。
+    *   各リリースでの性能向上を追跡するための公開ダッシュボード
+*   **CPU性能の向上**
+    *   継続した最適化と量子化したカーネル
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: The translation for `float` is missing.

File: site/ja/lite/guide/roadmap.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+    *   第一級のx86サポート
+*   **NN APIサポートの更新**
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: [Weak Suggestion]

I think `最新のNN APIをサポート` is better.

File: site/ja/lite/guide/roadmap.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+    *   新しいAndroid Q NN APIの機能、opsと型のフルサポート
+*   **GPUバックエンドの最適化**
+    *   Android上でのOpenCLとVulkanのサポート
+    *   Metal最適化のためのMetalとObjective-CのCocoaPods
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: [Weak Suggestion]

`Metal最適化` -> `Metalアクセラレータ`

File: site/ja/lite/guide/roadmap.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+*   **Hexagon DSPバックエンド**
+    *   Android P以前のデバイスへの最初のDSP高速化の最初のリリース
+
+## 最適化
+
+*   **量子化**
+    *   ハイブリッド型カーネル用のトレーニング後の量子化 -- [ローンチ済み](https://medium.com/tensorflow/introducing-the-model-optimization-toolkit-for-tensorflow-254aca1ba0a3){:.external}
+    *   (8b)固定小数点カーネル用のトレーニング後の量子化 -- [ローンチ済み](https://medium.com/tensorflow/tensorflow-model-optimization-toolkit-post-training-integer-quantization-b4964a1ea9ba){:.external}
+    *   (8b)固定小数点カーネル用の量子化トレーニング
+    *   (8b)固定小数点RNN用の事後と同時トレーニングAPIの拡張
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: [Very Weak Suggestion]

トレーニング中、トレーニング後のAPIを、固定小数点RNN用に拡張

File: site/ja/lite/guide/roadmap.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+    *   低ビット幅(< 8b)の固定小数点カーネル用の量子化トレーニング
+*   **刈り込み / スパース性**
+    *   規模に応じたトレーニング中のウェイトの刈り込み -- [ローンチ済み](https://medium.com/tensorflow/tensorflow-model-optimization-toolkit-pruning-api-42cac9157a6a){:.external}
+    *   スパース性モデルの実行のサポート
+
+## 可搬性
+
+*   **マイクロコントローラのサポート**
+    *   スピーチと画像識別のユースケースへの、何種類かの32ビットアーキテクチャへのサポート追加
+    *   視覚とオーディオのデータ用のサンプルコードとモデル
```
Comment (nuka137) [inline site/ja/lite/guide/roadmap.md]: [Weak Suggestion]

* `視覚` -> `画像`

**Comment by lamberta** [comment]
Same as https://github.com/tensorflow/docs/pull/1222
We are unable to accept this PR without signing the CLA. Closing for now

**Pull request closed without merging**